packages/story-editor/src/components/form/tags/input.js

```diff
@@ -116,8 +149,10 @@ function Input({
               onUndo(e);
             }
           }
-
-          if (['Comma', 'Enter', 'Tab'].includes(e.key)) {
+          if (e.key === 'ArrowDown' && suggestedTerms.length > 0) {
+            menuRef?.current?.firstChild?.focus();
+          }
+          if (['Comma', ',', 'Enter', 'Tab'].includes(e.key)) {
```

Comment:
💭 I think we should remove `Tab` from being used for submission. Currently it will submit the value and then focus an option in the autocomplete list.

<img src="https://user-images.githubusercontent.com/22185279/135680457-25fa6933-0b42-4f04-97ec-8cadff6a11ef.gif" height="300px" />

Guttenberg doesn't submit on `Tab` and that way we can avoid this awkward jump:
<img src="https://user-images.githubusercontent.com/22185279/135680598-53e318af-d140-4b40-80b4-b939aedd0aa2.gif" height="300px" />